splunk/splunk_container_template.tmpl

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: { { .PodName } }
+  name: {{ .PodName}}
```

Comment:
is it intentional that it has a space after the opening braces but not space before the closing?

Reply:
Yes; I'm aware it should work with a space at the end as well, but wanted this change to be as tiny as possible to merge quickly.

Reply:
I meant this:
```suggestion
  name: {{ .PodName }}
```
It's a smaller change than what you did.

Reply:
I'm aware, but you missed my point: this (non)issue occur again in this file and the kwok template.

Reply:
I don't know what you mean, but I'm not sure it's important that I do. Don't let me stop you from merging.

Reply:
Since I see that reviews are coming fast anyway, changed in all other places.

Reply:
I understand what you meant now :)